CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
-### unreleased
+### 0.4.0 (2020-09-15)
```

Comment:
I don't think this should end up in this diff.

Reply:
I accidentally branched off latest / 0.4.0 prerelease , guess this PR won't get into 0.4 anyway so can just wait until Pepe's commit get into master